src/GlobalStyles.js

```diff
@@ -11,6 +11,15 @@ export const GlobalStyles = createGlobalStyle`
     box-sizing: inherit;
   }
 
+  body {
+  font-family: 'Poppins', sans-serif;
+  margin: 0;
+  word-break: break-word;
+  background: ${({ theme }) => theme.colors.whisper};
+  padding: 16px;
```

Comment:
I don't think that padding is needed 👌

Reply:
It's useful to improve the overall look and readability, but there is indeed no padding in the design, so I'm removing it. Thanks for your vigilance :-)